## Setup

In [ ]:
import pandas as pd
pd.options.display.float_format = '{:.1f}'.format

In [ ]:
ufos_path = 'YOUR PATH HERE'
ufo_sightings = pd.read_csv(ufos_path)

<ipython-input-2-60ea0c9e0214>:2: DtypeWarning: Columns (5,9) have mixed types. Specify dtype option on import or set low_memory=False.
  ufo_sightings = pd.read_csv(ufos_path)


## Use the `value_counts` method to determine how many sightings in **Australia** (abbreviated as "au") were captured in the UFO sightings dataset:

In [ ]:
ufo_sightings['country'].value_counts()

country
us    65114
ca     3000
gb     1905
au      538
de      105
Name: count, dtype: int64

**The dataset contains 538 sightings in Australia.**

## Use the `value_counts` method to determine which **three cities** had the most sightings in the UFO sightings dataset:

In [ ]:
ufo_sightings['city'].value_counts()

city
seattle                                  525
phoenix                                  454
portland                                 374
las vegas                                368
los angeles                              353
                                        ... 
playa del cura (grand canaria island)      1
livermore/pleasonton (area)                1
hamburg (germany)                          1
cole camp                                  1
hamstead (hollyridge)                      1
Name: count, Length: 19900, dtype: int64

**Seattle, Phoenix, and Portland had the most sightings.**

## Use the `value_counts` method to determine how many sightings are associated with the most recent "date posted" value in the UFO sightings dataset:

In [ ]:
dates_posted = pd.to_datetime(ufo_sightings['date posted'], errors='coerce')
dates_posted.value_counts().sort_index(ascending=False)

date posted
2014-05-08    120
2014-05-02    178
2014-04-24    134
2014-04-18    152
2014-04-11    174
             ... 
1998-11-01    191
1998-09-26    160
1998-06-18     35
1998-06-02     50
1998-03-07    235
Name: count, Length: 317, dtype: int64

**120 sightings were posted on 05/08/2014**